zlp-scheduler/features/view_result_testing.feature

```diff
@@ -0,0 +1,29 @@
+Feature: An Admin can view the result
+  As an administrator
+  so that I can run the algorithm
+  I want to be able to view the result.
+  
+Background: log in as an admin
+  Given the following terms exist:
+  | name          |
+  | Test Term     |
+  And the active term is "Test Term"
+  And the following cohorts exist:
+  | name          |
+  | Test Cohort   |
+  And "Test Cohort" is in the current term
+  And I am logged in as an admin
+
+Scenario: View the view result page
+  When I click "Test Cohort"
+  Then I should see the Test Cohort page
+  And I click "Find Class Time"
```

Comment:
This is really minor, but the test would reads a little clearer if this `And` was a `When` instead. Not a critical change, though.